test/REQUIRE

```diff
@@ -1,2 +1,2 @@
-DataStructures
+DataStructures 0.5.2
```

Comment:
The bugfix is needed on 0.6, but is not actually being used. I suppose that's not what these minimum version bounds are for. I'll remove this commit.